.golangci.yml

```diff
@@ -87,10 +87,10 @@ linters-settings:
     ignore-words:
       - importas
   nolintlint:
-    allow-leading-space: true # don't require machine-readable nolint directives (i.e. with no leading space)
+    allow-leading-space: false # don't require machine-readable nolint directives (i.e. with no leading space)
     allow-unused: false # report any unused nolint directives
-    require-explanation: false # don't require an explanation for nolint directives
-    require-specific: false # don't require nolint directives to be specific about which linter is being skipped
+    require-explanation: true # don't require an explanation for nolint directives
+    require-specific: true # don't require nolint directives to be specific about which linter is being skipped
```

Comment:
stronger requirements applied

Reply:
I fixed the comments here https://github.com/signalfx/splunk-otel-go/pull/1571/commits/45f17ed104a5d0f2ffdc4ccab88ca718869b3474